templates/Dockerfile.apk.template

```diff
@@ -10,4 +10,8 @@ RUN apk add --no-cache bash git openssh-client runit jq
 
 STOPSIGNAL SIGTERM
 
-CMD ["/sbin/runsvdir", "-P", "/etc/service"]
+ENV RUNITDIR=/sbin
+
+COPY tugboat-entrypoint.sh /tugboat-entrypoint.sh
+
+CMD ["/tugboat-entrypoint.sh"]
```

Comment:
It feels a little odd to use the nomenclature of "entrypoint" for the script (though we do demarcate it as *tugboat-* entrypoint), given that we do not override the `ENTRYPOINT` provided by the original solr image. It's not really _the_ entrypoint so it seems it might better be called something else. This is a minor point I am making.

Reply:
Great point! What do you think about `runitd.sh` (since we are doing more than just handling child zombie (🧟 👶) processes in here

Reply:
I named this `runit-init.sh`. Better?

Reply:
> I named this `runit-init.sh`. Better?

Yes, that is very specific, and removes the entrypoint confusion. One might want to add some comments to the script to say exactly why it was necessary and how it works (for our future forgetful selves), but this is great, and the rename is good.